CMakeLists.txt

```diff
@@ -194,56 +194,8 @@ include(CPack)
 option(WITH_EXAMPLES "Compile the libLX example programs."  OFF)
 
 # Which language bindings should be built
-option(WITH_CSHARP   "Generate the C# language interface for libLX."     OFF)
```

Comment:
I'm guessing the reason all these language bindings have been deleted is because we haven't implemented them yet?